01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,13 @@
+number = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]
+number.each do |x|
```

Comment:
この課題が `1000` までの FizzBuzz を出力しなさいという内容だったら 1 から 1000 まで全部書かなければならなくなるので、プログラムに任せましょう。

```suggestion
number = 1..20
number.each do |x|
```

`..` を使って 範囲オブジェクトを作ることで 1 ~ 20 を取り出しながらループすることもできますし

```suggestion
20.times do |x|
```

Integer#times メソッドを使うと、指定した数だけ繰り返すことができます。

Reply:
コメントテストです